module5/Publication.py

```diff
@@ -0,0 +1,23 @@
+from abc import ABC, abstractmethod
+from datetime import datetime
+
+
+class Publication(ABC):
+    DATE_FORMAT = "%d/%m/%Y"
+    DATE_TIME_FORMAT = "%d/%m/%Y %H:%M"
+    PUBLICATION_TYPES = ["News", "Private Ad", "Arabic Text"]
```

Comment:
Can be enum. I think it will be more preferable while possible further development

Reply:
Fixed by enum.